us_web_design_standards.info

```diff
@@ -1 +1,66 @@
-name = U.S. Web Design Standards
+name        = US Web Design Standards
+description = Open source UI components and visual style guide to create consistency and beautiful user experiences across U.S. federal government websites.
+screenshot  = screenshot.png
+core        = 7.x
+engine      = phptemplate
+
+;;  remove some of the system (core) CSS files
+;;  we can always add these back in as we need them
+stylesheets[all][] = system.admin.css
+stylesheets[all][] = system.maintenance.css
+stylesheets[all][] = system.menus.css
+stylesheets[all][] = system.messages.css
+stylesheets[all][] = system.theme.css
+
+;;  remove some core module files
+;;  we can always add these back in as we need them
+stylesheets[all][] = aggregator.css
+stylesheets[all][] = block.css
+stylesheets[all][] = book.css
+stylesheets[all][] = color.css
+stylesheets[all][] = comment.css
+stylesheets[all][] = dashboard.css
+stylesheets[all][] = dblog.css
+stylesheets[all][] = field.css
+stylesheets[all][] = field_ui.css
+stylesheets[all][] = file.css
+stylesheets[all][] = filter.css
+stylesheets[all][] = forum.css
+stylesheets[all][] = help.css
+stylesheets[all][] = image.admin.css
+stylesheets[all][] = image.css
+stylesheets[all][] = locale.css
+stylesheets[all][] = menu.css
+stylesheets[all][] = node.css
+stylesheets[all][] = openid.css
+stylesheets[all][] = poll.css
+stylesheets[all][] = profile.css
+stylesheets[all][] = search.css
+stylesheets[all][] = shortcut.admin.css
+stylesheets[all][] = simpletest.css
+stylesheets[all][] = taxonomy.css
+stylesheets[all][] = tracker.css
+stylesheets[all][] = update.css
+stylesheets[all][] = user.css
+
+;; add our own CSS files from the standards folder
+stylesheets[all][] = assets/css/google-fonts.css
+stylesheets[all][] = assets/css/main.css
+stylesheets[all][] = assets/css/overrides.css
+
+;; we may need to remove jquery here and recommend jquery_update module to users
```

Comment:
Thanks for noting this. Perhaps share findings in https://github.com/18F/web-design-standards-drupal/issues/2?